workspaces/global-floating-action-button/packages/app/src/components/Root/Root.tsx

```diff
@@ -98,14 +98,14 @@ export const Root = ({ children }: PropsWithChildren<{}>) => {
           },
           {
             slot: Slot.BOTTOM_LEFT,
-            icon: <LibraryBooks />,
-            label: 'Api Docs',
-            toolTip: 'Api Docs',
+            icon: <ExtensionIcon />,
```

Comment:
nit: API is an acronym that should be in capitals

```suggestion
            icon: <ExtensionIcon />,
            label: 'APIs',
            toolTip: 'APIs',            
```